src/Team17/Main.java

```diff
@@ -1,76 +1,135 @@
-		 int []arr=new int[size];
-		 System.out.println("Fill the array elements:");
-		 for(int i=0;i<arr.length;++i)
-		 	{arr[i]=input.nextInt();}
-		 System.out.println("<<<Main Menu>>>");
-		 System.out.println("1-Most repeated value\t2-Sort\t3-Shuffle\t4-Find the largest prime\t5-Find the smallest prime\t6-Check palindrome");
-		 System.out.println("7-Check sorted\t8-Count Primes-\t9-Reverse array\t10-Shift array\t11-Distinct array\t12-Get the maximum 3 numbers");
-		 System.out.println("13-Get the minimum 3 numbers\t14-Get average\t15-Get median\t16-Return only primes\t17-Zero if less than zero\t18-Execute all functions");
-		 System.out.println("Enter the number of which function to be executed:");
-		 choice=input.nextInt();
-		 switch(choice)   //Add your functions 
-		 {
-		 case 1:  //Most repeated value
-			 break;
-		 case 2: //Sort
-			 break;
-		 case 3: //Shuffle
-			 break;
-		 case 4://Find the largest prime
-			 break;
-		 case 5://Find the smallest prime
-			 break;
-		 case 6: //Check palindrome
-			 break;
-		 case 7: //Check sorted
-			 break;
-		 case 8:
-		 {
-			 Count_Primes obj = null;
-			 System.out.println("The number of prime numbers:"+obj.Count_Primes(arr));
-		 }
-			 break;
-		 case 9://Reverse array
-			 break;
-		 case 10://Shift array
-			 break;
-		 case 11://Distinct array
-			 break;
-		 case 12://Get the maximum 3 numbers
-			 break;
-		 case 13://Get the minimum 3 numbers
-			 break;
-		 case 14://Get average
-			 break;
-		 case 15://Get median
-			 break;
-		 case 16://Return only primes
-			 break;
-		 case 17://Zero if less than zero
-			 break;
-		 case 18://option to perform all developed functions
-		 {
-			 //Add your function here also
-			 Count_Primes obj = null;
-			 System.out.println("The number of prime numbers:"+obj.Count_Primes(arr));
-		 }
-			 break;
-		default:
-		System.out.println("Invalid input!");
-		break;
+		System.out.println("Press 1 to input numbers and 2 for characters");
+		int typeOfArray = input.nextInt();
+		input = new Scanner(System.in); // if i don't re declare it raises some run time error
+		System.out.println("Please Input the Array");
+		String array = input.nextLine();    // get the entire line after the prompt
+
+		if (typeOfArray == 1) {
+			String[] numbers = array.replace(" ", "").split(",");
+			int[] a = new int[numbers.length];
+			for (int i = 0; i < numbers.length; i++) {
+				System.out.println(numbers[i]);
+				a[i] = Integer.parseInt(numbers[i]);
+			}
+		} else {
+			char[] char_array = array.toCharArray(); //i haven't test this well
+		}
+
+		print_messages();
+		int choice = input.nextInt();
+
+		while (choice > 0) {
+			if (choice == 18) {//this will loop on all functions from 1 to 17
+				for (int i = 1; i < 18; i++) {
+					switch_runner(i);
+				}
+			} else {
+				System.out.println("hekko");
+				switch_runner(choice);
+			}
+			print_messages();
+			 choice = input.nextInt();
+		}
+		
+	}
+
+	private static void print_messages(){
```

Comment:
I think the function should be above the main

Reply:
what's the difference as i test it and it's working